content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,207 @@
+|     | **Scale Min** | Override value for the minimum scale multiplier. | Float |
+|     | **Scale Max** | Override value for the maximum scale multiplier. | Float |
+| Altitude Filter |     | Settings for controlling per instance overrides for the altitude filter component. |     |
+|     | **Altitude Filter Override Enabled** | When enabled, the altitude filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Altitude Filter Min** | Override value for the minimum altitude accepted by the filter. | Float |
+|     | **Altitude Filter Max** | Override value for the maximum altitude accepted by the filter. | Float |
+| Distance Between Filter (Radius) |     | Settings for controlling per instance overrides for the distance between filter component. |     |
+|     | **Distance Between Filter Override Enabled** | When enabled, the distance between filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Bound Mode** | When performing distance checks between two instances, this setting determines whether the radius of the mesh asset is used vs a manually entered radius<br><br>Radius<br><br>MeshRadius. |     |
+|     | **Radius Min** | User defined radius for distance checks. | Float |
+| Surface Slope Alignment |     |     |     |
+|     | **Surface Slope Alignment Override Enabled** | When enabled, the slope alignment modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Surface Slope Alignment Min** | Override minimum alignment strength. | Float |
+|     | **Surface Slope Alignment Max** | Override maximum alignment strength. | Float |
+| Slope Filter |     |     |     |
+|     | **Slope Filter Override Enabled** | When enabled, the slope filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Slope Filter Min** | Override minimum slope accepted by filter. | Float |
+|     | **Slope Filter Max** | Override maximum slope accepted by filter. | Float |
+| Surface Mask Filter |     |     |     |
+|     | **Override Mode** | Controls how overrides are considered by the surface mask filter component<br><br>Disable - Overrides are completely ignored<br><br>Replace - Overrides replace those specified in the component<br><br>Extend - Overrides are added to the specified in the component. |     |
+|     | **Inclusion Tags** | A set of surface tags that dictate where vegetation can be placed. | SurfaceTagVector |
+|     | **Exclusion Tags** | A set of surface tags that dictate where vegetation will not be placed. | SurfaceTagVector |
+| Surface Mask Depth Filter |     |     |     |
```

Comment:
```suggestion
| Surface Mask Depth Filter |   |  Settings for controlling per-instance overrides for the [Surface Mask Depth Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-surface-mask-depth-filter) component.  |     |
```
Added this group property description because the individual properties lacked a link to the relevant component.